src/pages/leader-board/leader-board.page.jsx

```diff
@@ -0,0 +1,40 @@
+import { useState ,useEffect } from 'react';
+import "./leader-board.scss";
+import TranslationService from '../../services/translation.service';
+import AuthenticationService from '../../services/authentication.service';
+import { Endpoints } from '../../constants';
+const LeaderBoardPage = (props) => {
+  const leaderBoardPlayers = props.leaderBoardPlayers;
+    const authenticationService = new AuthenticationService();
+    const translationService = new TranslationService();
+    const [leaderboardUsers, setLeaderboardUsers] = useState([]);
+    const getAllLeaderBoard =() => {
+      authenticationService.get(Endpoints.usersTopLeaderboard).then((result) => {if(result.data){setLeaderboardUsers(result.data)}}).catch((error)=>{console.log(error);});
+    }
+    useEffect(() =>{
+      getAllLeaderBoard();
+    },[null]);
+
+  return (
+  <div className="qcg-leader-board-page qcg-flex qcg-flex-center">
+    <div className="wrapper">
+      <div className="board-wrapper qcg-flex qcg-flex-column qcg-flex-align-center">
+        <h1>{translationService.translate.leaderBoardPage.title}</h1>
+        {leaderboardUsers.length <= 10 ? (leaderboardUsers.map((top10, index)=>
+        {return <div className="circle-around qcg-flex qcg-flex-row qcg-flex-align-center" key={index}>
+          <div className="id">
+          {index+1}
+          </div>
+          <img src={top10.picture} alt="userProfilePicture" />
+          <div className="username"> {top10.firstName} </div>
+          <div className="points"> {top10.score}</div>
+          </div>})):(
+             <div></div>
+           )} 
```

Comment:
Remove the unneeded empty div with the above multiple opening and closing brackets.